feature-libs/customer-ticketing/components/customer-ticketing-reopen/customer-ticketing-reopen-dialog/customer-ticketing-reopen-dialog.component.ts

```diff
@@ -18,6 +23,20 @@ export class CustomerTicketingReopenDialogComponent
     if (this.form.invalid) {
       this.form.markAllAsTouched();
       FormUtils.deepUpdateValueAndValidity(this.form);
+    } else {
+      this.customerTicketingDetailsService.createTicketEvent(
+        this.prepareTicketEvent()
+      );
     }
   }
+
+  protected prepareTicketEvent(): TicketEvent {
```

Comment:
we can probably improve this by creating a 'form service', where we pass value to build the 'form' data. An example would be the `feature-libs/checkout/scheduled-replenishment/components/services/checkout-replenishment-form.service.ts`